### ISSUE TITLE
Can not install Lichdom mod because of Skitia NPC mod issue

### ISSUE THREAD

**Issue by rubyFeedback** [LCA-EET/Lichdom#6]
When trying to install the latest lichdom release, it says it is detecting
skitia NPC mod, in particular **helga**, but it can not find the file called:

xaldsoa.d

inside of that subdirectory where helga/ is.

Because of this problem, lichdom-mod does not seem to want to continue installation,
leading to (I think) a perma-loop where it keeps on trying to install it, but never able to
finish it because the file does not exist or can not be found (no clue why it can not be
found, probably because it is not there; not sure if I chose some unusual options during 
skitia NPC mod installation, I did not expect this to create other issues in other mods).

I don't know the scripting language used in the infinity engine, but
perhaps it is possible to do a "if file exists check"; but, even aside from
this, I believe lichdom mod should continue to install if an issue is
encountered like this (with other mods, that is), and give a prompt
to the player to skip it. Perhaps I missed that option but I don't seem
to be able to break out of the installation loop right now.

I just tested it (I have to go to another computer, as the one I am
writing this is my linux dev-machine), and after it encounters the
above problem with skitia mod, it tells me it will uninstall the 
installed files (from the lichdom mod, some ~1000 files or so) - and
then it tries again to install the whole mod. So this is indeed a
loop I am stuck in for the time being here.

An additional suggestion may be to add an "install cross-mod
banter" option. That way this would allow an easy work-around,
as I wanted to test the mod, and cross-mod content is nice to
have, but ultimately not as important as the mainstay game.

Anyway, this is just information - I'll probably wait until there is
a fix, as I don't want to start a new game without having installed
all mods I want to test.

**Comment by LCA-EET** [comment]
Thanks for reporting the issue. The missing file was included in the installation archive, but it was referenced using the old (EET) path structure. Please download the latest release to complete your installation.

**Comment by rubyFeedback** [comment]
The bug I reported here was fixed (thank you by the way, so I closed the issue
request here, as the primary cause is now gone; the rest was secondary, and
dev time is a limited resource, so I try to keep the issue trackers clean, if
possible.

I did, however had, encounter another problem - this time I
copy/pasted it onto a USB stick and transferred it here onto my primary
computer. I'll file a separate issue for that other bug, though.
